tsconfig.json

```diff
@@ -1,11 +1,9 @@
 {
-	"extends": "@sindresorhus/tsconfig",
+	"extends": "tsconfigs/nodejs-module",
 	"compilerOptions": {
+		"skipLibCheck": true,
```

Comment:
Required, else some of this project's `d.ts` files fail to compile. Which seems like an issue, itself. But a different issue.

https://github.com/sindresorhus/tsconfig/blob/v0.7.0/tsconfig.json#L24